articles/api/authentication/_multifactor-authentication.md

```diff
@@ -0,0 +1,608 @@
+# Multifactor Authentication
+
+In addition to username and password, you can ask users for an additional factor as proof of identity before issuing the requested tokens.
```

Comment:
@erjohnson This made sense when it was directly after [the Resource Owner Password section](https://auth0-docs-content-pr-6008.herokuapp.com/docs/api/authentication#resource-owner-password) but it doesn't now. You need to add something else before that and definitely refer (and cross link) to the Resource Owner Password grant (for the endpoints that it's applicable, some endpoints in this section if for authentication with MFA, and others for the management of MFA and you should make that clear in the intro)